components/tooltip/index.css

```diff
@@ -149,7 +153,7 @@ governing permissions and limitations under the License.
 .spectrum-Tooltip-tip {
   position: absolute;
 
-  height: var(--mod-tooltip-tip-block-size, var(--spectrum-tooltip-tip-block-size));
+  height: var(--mod-tooltip-tip-inline-size, var(--spectrum-tooltip-tip-inline-size));
```

Comment:
Should this be `block-size` rather than `height`?

Reply:
The latest commit updates the usages of width and height in there with their logical properties.

Reply:
If this applies to both axises, should it be `var(--mod-tooltip-tip-size, var(--spectrum-tooltip-tip-size))`?